docs/user-guide/global-configurations/deployment-charts.md

```diff
@@ -73,107 +97,145 @@ You can use the following variables in the helm template (all the placeholders a
 | **deploymentType** | Deployment strategy used in the pipeline |
 | **app** | Application's ID within the Devtron ecosystem |
 | **env** | Environment used to deploy the chart |
-| **appMetrics** | For the App metrics UI feature to be effective, include the `appMetrics` placeholder. |
+| **appMetrics** | For the App metrics UI feature to be effective, include the `appMetrics` placeholder. | -->
 
-> **For example**:
-> 
-> To create a template file to allow Devtron to only render the repository name and the tag from the CI/CD pipeline that you created, edit the `.image_descriptor_template.json` file as:
-> ```bash
-> {
->     "image": {
->	        "repository": "{{.Name}}",
->	        "tag": "{{.Tag}}"
->     }
-> }
-> ```
+### 3. Add app-values.yaml
 
-### 3. Package the custom chart in the `*.tgz` format
+In the root directory of your chart, Devtron expects an `app-values.yaml` file and validates whether the content of `values.yaml` file is present in `app-values.yaml` file or not. 
```

Comment:
why is app-values.yaml required? why my values.yaml should be repeated in app-values.yaml?

Reply:
Explained it in the new PR